src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1061,7 +1057,7 @@ public override Task<object> ExecuteScalarAsync(CancellationToken cancellationTo
         /// <summary>
         /// Executes the command text against the connection.
         /// </summary>
-        /// <param name="behavior">An instance of <see cref="CommandBehavior"/>.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
```

Comment:
@roji You did something wrong with docs.

Reply:
Oops, fixed.